sympy/series/tests/test_limits.py

```diff
@@ -883,3 +883,7 @@ def test_issue_19770():
     m = Symbol('m', nonzero=True)
     assert limit(cos(m*x), x, oo) == AccumBounds(-1, 1)
     assert limit(cos(m*x)/x, x, oo) == 0
+
+
+def test_issue_8229():
+    assert limit((x**Rational(1,4)-2)/(sqrt(x)-4)**Rational(2,3), x, 16) == 0
```

Comment:
There can be spaces around `-` and after `,` inside `Rational`.
So something like this:
`limit((x**Rational(1, 4) - 2)/(sqrt(x) - 4)**Rational(2, 3), x, 16) == 0`

Reply:
updated